media-gfx/iscan-plugin-gt-f720/iscan-plugin-gt-f720-0.1.1.2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib rpm
+
+DESCRIPTION="Epson Perfection V300 PHOTO scanner plugin for SANE 'epkowa' backend"
+HOMEPAGE="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+SRC_URI="amd64? ( https://download2.ebz.epson.net/iscan/plugin/gt-f720/rpm/x64/iscan-gt-f720-bundle-1.0.1.x64.rpm.tar.gz -> ${PN}-bundle.rpm.tar.gz )
+	x86? ( https://download2.ebz.epson.net/iscan/plugin/gt-f720/rpm/x86/iscan-gt-f720-bundle-1.0.1.x86.rpm.tar.gz -> ${PN}-bundle.rpm.tar.gz )"
```

Comment:
You're putting two different files into same file which is obviously wrong and will explode horribly on the other system you haven't tested. Also this is `1.0.1` now, so this ebuild is one horribly big WTF to me.